packages/story-editor/src/components/button/index.js

```diff
@@ -18,68 +18,23 @@
  * External dependencies
  */
 import { forwardRef } from 'react';
-import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
+import { Button, BUTTON_VARIANTS } from '@web-stories-wp/design-system';
 import { ArrowDown as ArrowDownIcon, More as MoreIcon } from '../../icons';
```

Comment:
It'd be great to ditch the `MoreIcon` here that is internal to the editor in favor of the one in the design system: https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-icons--all-icons 

You can get to it from: `import { Icons } from '@web-stories-wp/design-system';` `Icons.More`

Reply:
Actually all icon changes are made in another PR #8651.